src/pages/MainPage/MainPage.jsx

```diff
@@ -7,8 +7,21 @@ import OurGuidesSection from './OurGuidesSection';
 import ContactsSection from './ContactsSection';
 import TravelTips from './TravelTips';
 import MenuBottom from './MenuBottom';
+import { useStatistics } from '../../hooks/useStatistics';
 
 const MainPage = () => {
+  const { statistics, error, isLoading } = useStatistics();
+  const totalGuides = isLoading
+    ? '...'
+    : error
+      ? 'N/A'
+      : statistics?.totalGuides;
```

Comment:
```suggestion
      : statistics?.totalGuides ?? 0;
```

Reply:
That sounds great! It's not for nothing that you are my mentor